src/components/PasswordInput.js

```diff
@@ -1,5 +1,34 @@
 import React from "react";
+import styled from "@emotion/styled";
 
-export default function PasswordInput({ value }) {
-  return <input value={value} />;
+const Input = styled.input`
+  background-color: hsl(
+    ${props => (props.value?.length <= 10 ? props.value?.length * 10 : 100)},
+    100%,
+    50%
+  );
+`;
+
+const Inputt = styled.input`
+  background-color: hsl(
+    ${props => (if(props.value?.length === 0) {
+
+    })},
+    100%,
+    50%
+  );
+`;
+
+export default function PasswordInput() {
+  const [inputValue, setInputValue] = React.useState(null);
```

Comment:
I guess,  it's better to use an empty string as value  props on input...otherwise you may get an error 
React.useState("")